### ISSUE TITLE
Fix "bpversion" usage in RST documentation

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#4571]
The LaTeX tutorial used script https://github.com/biopython/biopython/blob/biopython-182/Doc/version.in to capture the current version of Biopython, run via https://github.com/biopython/biopython/blob/biopython-182/Doc/Makefile which passed this to the compiler by defining ``\bpversion`` via the command line.

This has been left in the RST conversion as the literal string ``\bpversion`` and is used to try to link to specific pages in the matching version of the API docs:

```
$ grep bpversion *.rst
chapter_cookbook.rst:`online <http://www.biopython.org/docs/\bpversion/api/Bio.SeqIO.QualityIO.html>`__):
chapter_msa.rst:`online <http://biopython.org/docs/\bpversion/api/Bio.AlignIO.html>`__):
chapter_pairwise2.rst:API <http://biopython.org/docs/\bpversion/api/Bio.pairwise2.html>`__.
chapter_quick_start.rst:documentation <http://biopython.org/docs/\bpversion/api/>`__) or
chapter_searchio.rst:BLAST <http://biopython.org/docs/\bpversion/api/Bio.SearchIO.BlastIO.html>`__
chapter_searchio.rst:`BLAT <http://biopython.org/docs/\bpversion/api/Bio.SearchIO.BlatIO.html>`__.
chapter_searchio.rst:`documentation <http://biopython.org/docs/\bpversion/api/Bio.SearchIO.html#module-Bio.SearchIO#module-Bio.SearchIO>`__
chapter_searchio.rst:`documentation <http://biopython.org/docs/\bpversion/api/Bio.SearchIO.html#module-Bio.SearchIO>`__
chapter_seq_annot.rst:`SeqRecord <http://biopython.org/docs/\bpversion/api/Bio.SeqRecord.html>`__
chapter_seq_annot.rst:`SeqFeature <http://biopython.org/docs/\bpversion/api/Bio.SeqFeature.html>`__):
chapter_seqio.rst:`online <http://biopython.org/docs/\bpversion/api/Bio.SeqIO.html>`__):
chapter_seqio.rst:`online <http://biopython.org/docs/\bpversion/api/Bio.SeqIO.html>`__).
```

As a stop-gap solution, we could replace ``\bpversion`` with ``dev`` or ``latest`` but I'd rather we fix this properly.

Since the Tutorial and the API documentation are now all combined into one, this can be done with a standard RST ``:ref:`` command instead I think.

**Comment by peterjc** [comment]
We might be able to use |version| or |release| in the RST? See ``conf.py`` and Sphinx docs.

**Comment by goutamdh** [comment]
Hi @peterjc ,

I want to work on this issue. can I pick ?

**Comment by peterjc** [comment]
Sure - do you know how to make references in RST?

Do you know how to setup and run Sphinx? The command ``make -C Doc/ html`` ought to work from the root of the repository. Or ``cd Doc/`` and then ``make html`` (or ``make latedpdf``).

**Comment by goutamdh** [comment]
That I will take a look

**Comment by peterjc** [comment]
Yes, this does still need fixing. I'd forgotten about it, and we really ought to fix this before the Biopython 1.84 release so please do have a go. Thank you!

**Comment by peterjc** [comment]
From local testing, ``:version:`` can be used in text but not within URLs.

All the ``\bpversion`` occurances are in fact within API links. We can partly fix this by using a relative URL ``../api/`` instead - which would work for the HTML output but not the PDF. The proper fix is to turn them into ``py:mod:...`` references since the Tutorial and API docs are now combined under Sphinx.

I am testing that now.
